k8s/wordpress/Makefile

```diff
@@ -99,6 +100,16 @@ app/build:: .build/wordpress/deployer \
 	@touch "$@"
 
 
+.build/wordpress/apache-exporter: .build/var/REGISTRY \
+                                  .build/var/TAG \
+                                  | .build/wordpress
+	docker pull marketplace.gcr.io/google/wordpress5-php7-apache:exporter
+	docker tag marketplace.gcr.io/google/wordpress5-php7-apache:exporter \
+	    "$(REGISTRY)/wordpress/apache-exporter:$(TAG)"
```

Comment:
I like the specificity of apache-exporter name - I am wondering if we shouldn't use the same tag for the image, so: marketplace.gcr.io/google/wordpress5-php7-apache:apache-exporter.

Reply:
Discussed offline, I'll keep this as it is for now.